proto/commonfate/control/config/v1alpha1/availability_spec.proto

```diff
@@ -21,13 +21,15 @@ message AvailabilitySpec {
     commonfate.entity.v1alpha1.EID role = 3;
     commonfate.entity.v1alpha1.EID target = 4;
     commonfate.entity.v1alpha1.EID identity_domain = 6;
+    optional int64 role_priority = 7;
```

Comment:
Lets change these to non optionals, that way they just default to 0
It makes the types easier to deal with in the backend and we treat 0 as unset for priority anyway.